freezefrog/__init__.py

```diff
@@ -1,17 +1,52 @@
-import calendar
 import datetime
+import time
+from unittest.mock import patch
 
 try:
-    from unittest.mock import patch
+    import pytz
 except ImportError:
-    from mock import patch
+    pass
 
-__all__ = ['FreezeTime']
+__all__ = ["FreezeTime"]
```

Comment:
Minor, but we usually use single quotes.

Reply:
Done in https://github.com/closeio/freezefrog/pull/13/commits/bd9f96126a0ede1b3a178f441d5b495b2d30d564.